backend/src/main/java/uk/ac/cam/cl/kilo/Server.java

```diff
@@ -2,53 +2,61 @@
 
 import java.io.*;
 import static spark.Spark.*;
-package uk.ac.cam.cl.kilo.database;
+import uk.ac.cam.cl.kilo.HttpRequest;
 
-public class Server extends HttpServlet {
+public class Server{
   public static void main(String args[]){
 
-    get("/facebook/register/stage1", (request, response) -> {
+     get("/facebook/register/stage1", (request, response) -> {
+      String state = RandomStringUtils.random(length, true, true);// random state
```

Comment:
We can't use this because it isn't cryptographically secure. Please `git pull` and use the secure random hex string generator method we already have.